Asterix/tests/test_coronagraph.py

```diff
@@ -18,7 +18,7 @@ def test_all_coronagraphs():
 
     # Set coronagraph to be tested
     coros_to_test = ["fqpm", "wrapped_vortex", "classiclyot", "knife", "hlc", "vortex"]
-    expected_attenuation = [1e-20, 3e-9, 5e-3, 1e-2, 1e-3, 5e-9]  # Note that these are for the 200 px pupil
+    expected_attenuation = [1e-20, 1e-8, 5e-3, 1e-2, 1e-3, 5e-9]  # Note that these are for the 200 px pupil
```

Comment:
Where does this performance requirement decrease in this test come from? Did the test start failing with the changes in this PR?

Reply:
yeah, the regionnal sampling is so sensitive to these stuff that the change of Lyot size / FP resolution juste change the contrast by a factor of 4. I found a new point that seems to give 2.10-9 but this is just stupid to have that amount in the test because you cannot change your parameters.

Reply:
I don't understand. Do you mean the performance is very sensitive to one parameter for all coronagraphs? for one specific coronagraph (wrapped vortex)? Which parameter is it (size of the pupil in pixels?)?

Reply:
This is only for the wrapped vortex. There are "hidden parameters" for the regional sampling: the size of the focal plane and the samplings of the regions (usually 2 different sampling + the outer one which is always nyquist in our case). for a given size of entrance pupil (for example 200x200), I found the smallest parameter that allow you to get under a certain limit in contrast (see array below)
<img width="891" alt="Screenshot 2023-04-28 at 07 49 29" src="https://user-images.githubusercontent.com/22528508/235065139-6d10121c-1723-4a0c-9d87-dc9b35ba2067.png">

However, because I really tried to take them as small as possible (to go faster), they are very sensitive and can increase the simulated contrast if you change the type of corono (vortex or wrapped vortex) + size of Lyot + resolution in the focal plane, pupil size, etc. So we can either use a very oversized focal plane size (=1024), which will always give ok contrast but will be long or just work with higher contrast by default and optimize these number in specific case when you want to optimize your simulation. I don't think I should take time to optimize these number just for the testing each time

Reply:
I just wanted to clarify due to which parameter change the test contrast decreased, since not all that much changed in the relevant parameters. Is it due to the new Lyot stop diameter?